src/Microsoft.AspNetCore.Razor.VSCode/src/RazorDocumentManager.ts

```diff
@@ -128,9 +128,10 @@ export class RazorDocumentManager implements IRazorDocumentManager {
         return document;
     }
 
-    private openDocument(uri: vscode.Uri) {
+    private async openDocument(uri: vscode.Uri) {
```

Comment:
Why async? There is nothing async going on here.

Reply:
Oh derp, forgot that I had removed the async pieces when investigating the fix 😲